tokio/src/runtime/builder.rs

```diff
@@ -47,25 +45,22 @@ pub struct Builder {
     num_threads: usize,
 
     /// Name used for threads spawned by the runtime.
-    thread_name: String,
+    pub(super) thread_name: String,
 
     /// Stack size used for threads spawned by the runtime.
-    thread_stack_size: Option<usize>,
+    pub(super) thread_stack_size: Option<usize>,
 
     /// Callback to run after each thread starts.
     after_start: Option<Callback>,
 
     /// To run before each worker thread stops
     before_stop: Option<Callback>,
-
-    /// The clock to use
-    clock: timer::Clock,
```

Comment:
`Clock` is no longer configurable, but with feature `test-util`, it can be controlled.